src/imputors/standardize.jl

```diff
@@ -0,0 +1,84 @@
+"""
+    Standardize(; values)
+
+Standardize (or replace) various missing data values with `missing`.
+This is useful for downstream imputation methods that assume missing data is represented by
+a `missing`.
+
+Warning: In-place methods are only applicable for datasets which already `allowmissing`.
+
+# Keyword Arguments
+* `value::Tuple`: A tuple of values that should be considered `missing`
+
+# Example
+```jldoctest
+julia> using Impute: Standardize, impute
+
+julia> M = [1.0 2.0 -9999.0 NaN 5.0; 1.1 2.2 3.3 0.0 5.5]
+2×5 Array{Float64,2}:
+ 1.0  2.0  -9999.0  NaN    5.0
+ 1.1  2.2      3.3    0.0  5.5
+
+julia> impute(M, Standardize(; values=(NaN, -9999.0, 0.0)))
+2×5 Array{Union{Missing, Float64},2}:
+ 1.0  2.0   missing  missing  5.0
+ 1.1  2.2  3.3       missing  5.5
+```
+"""
+struct Standardize <: Imputor
```

Comment:
This is the least effective Imputor.
Infact it makes more missings.

Reply:
https://github.com/invenia/Impute.jl/issues/77